scss/forms/_meter.scss

```diff
@@ -37,7 +37,13 @@ $meter-fill-bad: $alert-color !default;
     height: $meter-height;
     margin-bottom: 1rem;
 
-    appearance: none;
+    // Specific Hack to disable `-webkit-appearance: none` from getting prefixed,
+    // We have disabled autoprefixer first and are just using `-moz-appearance: none` 
+    // as a prefix and neglecting `-webkit-appearance: none`
```

Comment:
That's not a hack, nor in Sass, nor in CSS. We simply do not write `-webkit-appearance`. But you can explain _why_: not necessary in Chrome, and create a bug (with a link to the issue).

However, only do this if we are sure that we _need_ appearance:none in other browsers (Chrome, Firefox, IE, Edge, Opera...)